evm/src/test/java/org/hyperledger/besu/evm/operation/Create2OperationTest.java

```diff
@@ -185,6 +185,7 @@ void shouldCalculateAddress(
       final int ignoredExpectedGas) {
     setUp(sender, salt, code);
     final Address targetContractAddress =
+        operation.targetContractAddress(
```

Comment:
what is the significance of this change?

Reply:
I didn't make any changes there.

Reply:
can you revert this change - may be a result of resolving a merge conflict? I'm surprised it doesn't cause a compilation error actually

Reply:
> can you revert this change - may be a result of resolving a merge conflict? I'm surprised it doesn't cause a compilation error actually

Alright, I will in my next push.

Reply:
> can you revert this change - may be a result of resolving a merge conflict? I'm surprised it doesn't cause a compilation error actually

I checked the main and compared it with mine. It seems to be the same. I am not sure how to go about reverting this.

Reply:
it's causing a compilation error for me - I just tried to run spotlessApply 

problem in 'src/test/java/org/hyperledger/besu/evm/operation/Create2OperationTest.java':
190:91: error: ')' or ',' expected
com.google.googlejavaformat.java.FormatterException: 190:91: error: ')' or ',' expected
        at com.google.googlejavaformat.java.FormatterException.fromJavacDiagnostics(FormatterException.java:51)
        at com.google.googlejavaformat.java.RemoveUnusedImports.parse(RemoveUnusedImports.java:264)
        at com.google.googlejavaformat.java.RemoveUnusedImports.removeUnusedImports(RemoveUnusedImports.java:218)
        at com.diffplug.spotless.glue.java.GoogleJavaFormatRemoveUnusedImporterFormatterFunc.apply(GoogleJavaFormatRemoveUnusedImporterFormatterFunc.java:38)

Reply:
> it's causing a compilation error for me - I just tried to run spotlessApply
> 
> problem in 'src/test/java/org/hyperledger/besu/evm/operation/Create2OperationTest.java': 190:91: error: ')' or ',' expected com.google.googlejavaformat.java.FormatterException: 190:91: error: ')' or ',' expected at com.google.googlejavaformat.java.FormatterException.fromJavacDiagnostics(FormatterException.java:51) at com.google.googlejavaformat.java.RemoveUnusedImports.parse(RemoveUnusedImports.java:264) at com.google.googlejavaformat.java.RemoveUnusedImports.removeUnusedImports(RemoveUnusedImports.java:218) at com.diffplug.spotless.glue.java.GoogleJavaFormatRemoveUnusedImporterFormatterFunc.apply(GoogleJavaFormatRemoveUnusedImporterFormatterFunc.java:38)

This should be fixed now.